projects/dotcms-field-elements/src/models/dot-content-type-field.model.ts

```diff
@@ -1,6 +1,6 @@
 export interface DotCMSContentTypeField {
     dataType?: string;
-    defaultValue?: string;
+    defaultValue?: string | any;
```

Comment:
the `defaultValue` can only be set by do `<dot-key-value defaultValue="xxxxxx" />` right? is so, why any? and if we need any, we could use `{[key: string]: string}`.